hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/UnderReplicatedProcessor.java

```diff
@@ -119,12 +119,14 @@ private void adjustPendingOps(ContainerID containerID, SCMCommand<?> cmd,
       for (int i = 0; i < targetIndexes.length; i++) {
         pendingOps.scheduleAddReplica(containerID, targets.get(i),
             targetIndexes[i]);
+        replicationManager.getMetrics().incrEcReconstructionCmdsSentTotal();
       }
     } else if (cmd.getType() == StorageContainerDatanodeProtocolProtos
         .SCMCommandProto.Type.replicateContainerCommand) {
       ReplicateContainerCommand rcc = (ReplicateContainerCommand) cmd;
       pendingOps.scheduleAddReplica(
           containerID, targetDatanode, rcc.getReplicaIndex());
+      replicationManager.getMetrics().incrEcReplicationCmdsSentTotal();
```

Comment:
Might need to the `if replicaIndex > 0 ... else if replicaIndex == 0` logic here, but not at line 122 above, as it is only EC code that can get there.

Reply:
Makes sense. Will update the PR